components/List/examples/MDCSelfSizingStereoCellExample.m

```diff
@@ -30,7 +30,7 @@ @interface MDCSelfSizingStereoCellExample () <UICollectionViewDelegate, UICollec
 @property(nonatomic, strong) UICollectionView *collectionView;
 @property(nonatomic, strong) UICollectionViewFlowLayout *collectionViewLayout;
 @property(nonatomic, strong) NSArray *randomStrings;
-@property(nonatomic, strong) MDCListScheme *listScheme;
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
```

Comment:
Nit: Please change this to type `id<MDCContainerScheming>` to match the catalog's expected convention.